microsoft-edge/extensions-chromium/getting-started/manifest-format.md

```diff
@@ -0,0 +1,207 @@
+---
+description: Learn about the format of the manifest file in an extension package.
+title: Manifest file format
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 06/07/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, web development, html, css, javascript, developer, extensions, mv2, mv3, manifest
+---
+
+# Manifest file format
+
+Every extension for Microsoft Edge (Chromium) has a JSON-formatted manifest file, named `manifest.json`.  The manifest file is the blueprint of your extension.  The manifest file includes information such as:
+
+*  The version number of the extension.
+*  The title of the extension.
+*  The permissions that are needed for the extension to run.
+
+The format for `manifest.json` for extensions is moving from Manifest V2 to Manifest V3.  Both formats are shown here.  To migrate a Manifest V2 extension to Manifest V3, navigate to [Prepare to update your extensions from Manifest v2 to v3][MigrateToMV3].
+
+
+## Format of manifest\.json for extensions using Manifest V3
+
+The following pseudocode shows the fields that are supported in `manifest.json` for extensions, for a Manifest V3 package.
+
+```json
```

Comment:
You can see my minor tweaks to the JSON listings at https://github.com/MicrosoftDocs/edge-developer/pull/1267/commits/0587d65abc99089bb8de8e252392a29a782de51b